src/hugo/content/en-us/patterns/MVC/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Model-View-Controller Device Software Design"
+weight: 10
+aliases:
```

Comment:
This section can be removed. Aliases can be added later if the site is refactored.